src/backend/core/api/filters.py

```diff
@@ -1,18 +1,73 @@
 """API filters for Impress' core application."""
 
+import unicodedata
+
+from django.db.models import CharField, Func
 from django.utils.translation import gettext_lazy as _
 
 import django_filters
 
 from core import models
 
 
+def remove_accents(value):
+    """Remove accents from a string (vélo -> velo)."""
+    return "".join(
+        c
+        for c in unicodedata.normalize("NFD", value)
+        if unicodedata.category(c) != "Mn"
+    )
+
+
+# pylint: disable=abstract-method
+class Unaccent(Func):
+    """
+    PostgreSQL unaccent function wrapper for use in Django ORM queries.
+
+    This allows you to annotate a field using the unaccented version of a
+    text column, enabling accent-insensitive filtering.
+    """
+
+    function = "unaccent"
```

Comment:
This function already exists in django https://docs.djangoproject.com/en/5.1/ref/contrib/postgres/lookups/#unaccent

Reply:
True! Good catch!